adrs/0011-annotations-lifespan.md

```diff
@@ -0,0 +1,28 @@
+# 10. Annotation types
+Date: 2022-07-05
+
+## Status
+Accepted
+
+## Context
+Not all annotations are needed in all stages of AMF (parsing, resolution/transformation, validation, rendering/emission).
+
+## Decision
+For this reason, the decision has been made in favor of adding 3 distinct types of annotations, that all others should implement:
+
+- `ResolvableAnnotation`
+    - An annotation that must be resolved
+    - To allow deferred resolution on unordered graph parsing
```

Comment:
This phrase is pretty but it isn't clear. What is "deferred resolution" ? When is graph parsing "unordered" ?

Reply:
We all agree that graph our graph do not ensure order, but I think that fact has no relation with the definition of a resolvable annotation.
For me, resolvable annotations means that the value of the annotation must be processed into the rdf model while parsing it.